hooks/useTBAPurchase.tsx

```diff
@@ -16,6 +16,18 @@ const useTBAPurchase = () => {
   const [loading, setLoading] = useState(false)
   const { isLoggedByEmail } = useUserProvider()
   const { prepare } = usePreparePrivyWallet()
+  const [preparedMulticalls, setPreparedMulticalls] = useState(null)
+
+  useEffect(() => {
+    const init = async () => {
+      const prepared = await getPreparedMulticalls(
+        (connectedWallet as `0x${string}`) || "0x0000000000000000000000000000000000000000",
```

Comment:
can you update to use `zeroAddress` from `viem`?
- replace hard-coded magic address 🪄 
```suggestion
        (connectedWallet as `0x${string}`) || zeroAddress,
```

Reply:
fixed ✅